getting-started-github-graphql-api.md

```diff
@@ -40,59 +42,59 @@ When interacting with GitHub's GraphQL API, it would be great to fill your own a
+You can also create repositories on GitHub. In the words of their official glossary: *"A repository is the most basic element of GitHub. They're easiest to imagine as a project's folder. A repository contains all of the project files (including documentation), and stores each file's revision history. Repositories can have multiple collaborators and can be either public or private."* {{% a_blank "GitHub's glossary" "https://help.github.com/articles/github-glossary/" %}} will explain the key terms--repository, issue, clone, fork, push--which are necessary to follow along with the upcoming sections. Basically a repository is the place for application source code that can be shared with others.  I encourage you to put a few of your projects into GitHub repositories, so you can access them all later with what you've learned about their GraphQL API
 
-If you don't have any projects for uploading (pushing) them as your repositories, you can also fork repositories from other GitHub users to continue working on them. After you have forked a repository, it is available in your list of repositories. It is basically a clone of the original repository where you can add your specific changes without chaning the original project. That's how open source works after all. There are many public repositories on GitHub which can be cloned to your local machine or forked to your list of repositories on GitHub in order to contribute to them. For instance, when you visit {{% a_blank "my GitHub profile" "https://github.com/rwieruch" %}}, you can see all my public repositories (not all of these are mine, because some of them are simply forked from others). You can navigate to a few of those repositories and fork them. Then you have those repositories accessible in your GitHub account which become available via GitHub's GraphQL API later on.
+If you don't have any projects to upload, you can always 'fork' repositories from other GitHub users and work on copies of them. A fork is basically a clone of a repository where you can add changes without altering the original. There are many public repositories on GitHub that can be cloned to your local machine or forked to your list so you can get an understanding of the mechanic through experimentation. For example, if you visit {{% a_blank "my GitHub profile" "https://github.com/rwieruch" %}}, you can see all my public repositories, though not all of these are mine, because some of them are just forks of others.  Feel free to fork these repositories if you'd like to use them as practice, and if you'd like them to be accessible via GitHub's GraphQL API later.
```

Comment:
"... and if you'd like them to be accessible via GitHub's GraphQL API later." to "and if you'd like them to be accessible via GitHub's GraphQL API on your account later."

Reply:
Changed to: "....if you'd like them to be accessible via GitHub's GraphQL API from your own account."  Here, leave it up to the reader to decide if they want to experiment now or later.